docs/source/using/troubleshooting.adoc

```diff
@@ -11,6 +11,26 @@ toc::[]
 
 This section contains solutions to common problems that you might encounter while using Minishift.
 
+[[minshift-update-failed-due-to-permission-denied]]
+== Permission cause update to fail
+
+Depending on your operating system if you have minishift binary installed in location where current user does not have write permission then `minishift update` will fail.
+
+Workaround: When updating minishift use `sudo minishift update` (Linux/Unix).
```

Comment:
Workaround: Please provide the user with necessary access permission or you can run the user in an elevated permission mode e.g. `sudo minishift update` in GNU/Linux.